starter-code/js/index.js

```diff
@@ -1,52 +1,45 @@
-function deleteItem(e){
-
-}
-
-function getPriceByProduct(itemNode){
-
-}
-
-function updatePriceByProduct(productPrice, index){
-
-}
-
-function getTotalPrice() {
-
-}
-
-function createQuantityInput(){
-
-}
-
-function createDeleteButton(){
-
-}
-
-function createQuantityNode(){
-
-}
+var getProductPrice = function(productNode) {
+  var priceNode = productNode.querySelector(".price");
+  var price = priceNode.innerText;
+  return price;
+};
 
-function createItemNode(dataType, itemData){
+var getQuantity = function(productNode) {
+  var quantityNode = productNode.querySelector(".quantity");
+  var quantityInputBox = quantityNode.querySelector("input");
+  var quantity = quantityInputBox.value;
+  return quantity;
+};
 
-}
+var getTotal = function(price, quantity) {
+  var priceTotal = price * quantity;
+  return priceTotal;
+};
 
-function createNewItemRow(itemName, itemUnitPrice){
+var displayTotalPrice = function(productNode, price) {
+  var totalPriceNode = productNode.querySelector(".total-price");
+  totalPriceNode.innerText = price;
+};
 
-}
+var calculatePrices = function(productNode) {
+  var calculateBtn = productNode.querySelector(".btn-success");
+  addEventListener.calculateBtn("click", function() {});
+};
 
-function createNewItem(){
+window.onload = function() {
```

Comment:
the only thing that runs in your code is what you have in here. All the functions you have defined are not called.